backend/src/apiserver/main.go

```diff
@@ -72,12 +82,58 @@ func main() {
 		template.Launcher = common.GetStringConfig(launcherEnv)
 	}
 
-	clientManager := cm.NewClientManager()
+	logLevel := *logLevelFlag
+	if logLevel == "" {
+		logLevel = "info"
+	}
+
+	level, err := log.ParseLevel(logLevel)
+	if err != nil {
+		log.Fatal("Invalid log level:", err)
+	}
+	log.SetLevel(level)
+
+	zapLevel, err := common.ParseLogLevel(logLevel)
+	if err != nil {
+		glog.Infof("%v", err)
```

Comment:
I was thinking here would be the code that sets the default log level to info rather than assuming `ParseLogLevel` always returns an info level when there is an error.